index.yaml

```diff
@@ -86,7 +86,16 @@ entries:
     version: 1.0.0
   chart-operator:
   - apiVersion: v1
-    created: "2020-03-11T08:41:16.641646053Z"
+    appVersion: 1.0.0-alpha.1
```

Comment:
Can we place it one step downward or put this in the last? 
Otherwise, appcatalog.GetLatestChart would think that this is the last version of chart-operator.

Reply:
Good point. Thanks for fixing.